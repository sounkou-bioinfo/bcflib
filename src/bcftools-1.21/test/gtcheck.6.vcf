##fileformat=VCFv4.3
##reference=ref.fa
##contig=<ID=1,assembly=b37,length=107349540>
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	A	B	C	D
1	1	.	C	T	.	PASS	.	GT	0/0	0/0	0/1	0/1
