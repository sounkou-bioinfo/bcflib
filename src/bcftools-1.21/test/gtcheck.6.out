DCv2	B	C	1	8.675006e-01	2	1
DCv2	B	D	2	0.000000e+00	2	0
